lib/inferno/apps/cli/execute/abstract_outputter.rb

```diff
@@ -0,0 +1,41 @@
+module Inferno
+  module CLI
+    class Execute
+      # Subclass AbstractOutputter to implement your own outputter
+      #
+      # @abstract
+      # rubocop:disable Lint/UnusedMethodArgument
+      class AbstractOutputter
```

Comment:
I don't think we should add abstract classes. If you think we need some sort of really basic output class, make this some sort of null output class that works but doesn't print anything.